otri/filtering/filter.py

```diff
@@ -7,73 +7,73 @@ class Filter:
     Abstract class that defines an atom manipulation filter.
 
     Attributes:
-        input_streams : Collection[Stream]
-            Streams where to dequeue atoms from.
-        output_streams : Collection[Stream]
-            Streams where manipulated atoms are placed.
+        input : Sequence[str]
+            Name for input streams. If there are multiple streams the filter must define the right order.
+            Streams will be gathered inside the FilterList's dictionary of streams.
+        output : Sequence[str]
+            Name for output streams,
     '''
 
-    def __init__(self, input_streams: Collection[Stream], input_streams_count: int = 0, output_streams_count: int = 0):
+    def __init__(self, input: Sequence[str], output: Sequence[str], input_count: int = 0, output_count: int = 0):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'input'](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5649486)